spaces/S000198/properties/P000133.md

```diff
@@ -0,0 +1,12 @@
+---
+space: S000198
+property: P000133 # LOTS
+value: false
+refs:
+- mathse: 4915404
+  name: Why isn't every subspace of $\mathbb R$ a linearlly ordered topological space (LOTS)?
+- wikipedia: Order_topology
+  name: Order topology
```

Comment:
To follow the pattern for other wikipedia references: "Order topology on Wikipedia"

Reply:
```suggestion
  name: Order topology on Wikipedia
```